test/chunk.cc

```diff
@@ -1,7 +1,7 @@
 #include <gtest/gtest.h>
 
 extern "C" {
-#include "../src/chunk.h"
+#include "chunk.h"
```

Comment:
Then the test could have the user facing styled include like

```
#include <ddnet_protocol/chunk.h>
```

While for the internal .c files i would still keep just "chunk.h" so it looks like a file next to it even if it is not.